tests/model_forms/tests.py

```diff
@@ -739,8 +739,10 @@ def test_abstract_inherited_unique(self):
         DerivedBook.objects.create(title=title, author=self.writer, isbn=isbn)
         form = DerivedBookForm({'title': 'Other', 'author': self.writer.pk, 'isbn': isbn})
         self.assertFalse(form.is_valid())
-        self.assertEqual(len(form.errors), 1)
+        self.assertEqual(len(form.errors), 2)
         self.assertEqual(form.errors['isbn'], ['Derived book with this Isbn already exists.'])
+        self.assertEqual(form.errors['__all__'],
+                         ['Derived book with this Suffix1 and Suffix2 already exists.'])
```

Comment:
I'm not sure on this. It seems it could be unexpected (or at least backwards incompatible) to silently ignore the empty POST data for the suffix fields and use the model defaults instead. I think the usual case for setting `blank=True` or `required=False` on a non-null field is to allow setting the value manually during form cleaning or during `model.save()`.